pkg/time/time.go

```diff
@@ -16,10 +16,25 @@ func Open(l *lua.State) {
 
 var timeLibrary = []lua.RegistryFunction{
 	{"now", now},
+	{"nowUTC", nowUTC},
 	{"sleep", sleep},
 	{"since", since},
+	{"addTime", addTime}
```

Comment:
This should be `add`. It would be most flexible if it took a table as its second argument, representing the duration to add with optional fields for each of Go's [`time.Duration`](https://golang.org/pkg/time/#Duration) constants. E.g.
```lua
local t = time.now()
t = time.add(t, {hour=1, minute=3})
local s = time.format(s, "2006-01-02T15:04:05Z07:00" "UTC") -- RFC3339
```
A more flexible and "natural" alternative would be to return a UserData from `time.now()` that wraps a Go `time.Time` and provides a metatable with all the interesting methods, including `+`. That's likely a bunch more work for dubious value.